src/System.Net.Security/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -39,8 +39,8 @@ public void EventSource_EventsRaisedAsExpected()
                     {
                         // Invoke tests that'll cause some events to be generated
                         var test = new SslStreamStreamToStreamTest_Async();
-                        test.SslStream_StreamToStream_Authentication_Success();
-                        test.SslStream_StreamToStream_Successive_ClientWrite_Sync_Success();
+                        test.SslStream_StreamToStream_Authentication_Success().ConfigureAwait(false).GetAwaiter().GetResult();
+                        test.SslStream_StreamToStream_Successive_ClientWrite_Sync_Success().ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
We generally don't use ConfigureAwait in tests.  And in this case specifically, this is all being run in another process (due to the RemoteInvoke), and there won't be a sync ctx or custom scheduler configured there, so it'll also be a nop.  Did you find this actually helped, or it was just speculative?

Reply:
It was just speculative. I can remove it